about.css

```diff
@@ -0,0 +1,430 @@
+}
+
+.address-2 {
+  font-weight: 600;
+  color: #272a32;
+  text-decoration: underline;
+  margin-top: -10px;
+  padding-bottom: 30px;
+}
+
+h4 {
+  font-size: 15px;
+  padding: 0 90px;
+}
+
+.guide {
+  border-bottom: 1px solid orangered;
+  width: 10%;
+  margin: auto;
+}
+
+.logo {
+  border: 1px solid lightgrey;
+  height: 30%;
+  width: 90%;
+  padding: 0 80px;
+  margin: 20px 0;
+}
+
+h3 {
+  font-size: 20px;
+  padding-top: 30px;
+  border-top: 1px solid lightgray;
+}
+
+.outreach {
+  margin: 10px 30px;
+  width: 80%;
+}
+
+.partner {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-color: #272a32;
+  height: 25vh;
+}
+
+.guide- {
+  width: 1.5rem;
+  margin-top: 1rem;
+  border-bottom: 2px solid #ec5242;
+}
+
+li {
+  width: 30%;
+  margin: 10px;
+}
+
+.ul-logo {
+  background: #272a32;
+}
+
+.ul-logo li {
+  object-fit: cover;
+  filter: grayscale(1);
+}
+
+h5 {
+  color: white;
+  font-size: 20px;
+  padding-top: 20px;
+  padding-bottom: 10px;
+  margin-bottom: -1px;
+}
+
+.partner h5 {
+  margin-top: -1rem;
+}
+
+.logo-2 {
+  width: 40%;
+}
+
+.logo-3 {
+  margin-top: -3rem;
+  width: 40%;
+}
+
+.logo-1 {
+  width: 40%;
+}
+
+ul {
+  list-style: none;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  padding: 0;
+  margin-top: 20px;
+}
+
+.lagtechfest-logo {
+  width: 20%;
+  margin-top: 6rem;
+}
+
+footer {
+  display: flex;
+  justify-content: center;
+  margin: 0 10px;
+}
+
+.paragraph {
+  background-color: white;
+  color: #272a32;
+  font-size: 10px;
+  line-height: 18px;
+  margin-top: 80px;
+}
+
+#desktop-menu {
+  display: none;
+}
+
+.conference {
+  display: flex;
+  position: relative;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  margin: 2rem 0;
+  z-index: 1;
+  width: calc(100% - 2rem);
+  min-height: 320px;
+  text-align: center;
+  gap: 1rem;
+  cursor: pointer;
+}
+
+.conference img {
+  position: absolute;
+  top: 0;
+  left: 1rem;
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
+  z-index: -2;
+}
+
+.bg-overlay {
+  position: absolute;
+  top: 0;
+  left: 1rem;
+  width: 100%;
+  height: 100%;
+  background-color: #d72f1c;
+  opacity: 0.75;
+  z-index: -1;
+}
+
+.conference h4,
+h6 {
+  margin-top: 0;
+  color: #fff;
+}
+
+.conference h4 {
+  font-size: 2.5rem;
+  font-weight: 900;
+  margin-bottom: 0;
+  margin-bottom: 0.5rem;
+}
+
+.conference h6 {
+  font-size: 1.15rem;
+  font-weight: 600;
+  margin-bottom: 0;
+}
+
+@media (min-width: 768px) {
+  #menu {
+    display: none;
+  }
+
+  #desktop-menu {
+    display: block;
+  }
+
+  .menu-nav {
+    display: none;
+  }
+
+  ul {
+    list-style: none;
+  }
+
+  li {
+    width: 10%;
+    margin-bottom: 5px;
+    margin-right: 0;
+    margin-left: 0;
+  }
+
+  .navigation {
+    display: flex;
+    flex-wrap: wrap;
+    background-color: rgb(47, 47, 47);
+    justify-content: flex-end;
+    margin: 0%;
+    height: 5vh;
+    padding-right: 50px;
+  }
```

Comment:
- [ ] [OPTIONAL] I can see that you have a height that breaks the content of the card from being flexible and therefore crosses the border. Therefore I suggest, removing the height and considering using padding to create the matching space ☺️.

Your design 😎 
![image](https://user-images.githubusercontent.com/99394374/226098770-539ac479-7209-48a1-a966-59273b8d3fc9.png)